build/IPDK_Container/scripts/build_p4c.sh

```diff
@@ -33,7 +33,7 @@ git clone https://github.com/p4lang/p4c.git --recursive P4C
 cd P4C
 git checkout $P4C_SHA
 mkdir build && cd build
-cmake -DENABLE_BMV2=OFF -DENABLE_EBPF=OFF -DENABLE_UBPF=OFF \
+cmake -DENABLE_BMV2=OFF -DENABLE_EBPF=ON -DENABLE_UBPF=OFF \
```

Comment:
By default, option "DENABLE_EBPF" is ON.  I think it is better to remove the complete option "DENABLE_EBPF" from the **_cmake_** command.  If you observe, **_cmake_** option added only if we need a value other than default.

Reply:
Correct, thats how I compile my own p4c containers. DENABLE_* for different targets is on by default, only need to define when you don't want that target build into the compiled code.

Reply:
Will update and merge, thanks!